### PR TITLE
Check on error before close the expecter

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Only tests fixes

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
retest this please

**Comment by cynepco3hahue** [comment]
retest this please

**Comment by cynepco3hahue** [comment]
retest this please

File: tests/storage_test.go [hunk 1]
```diff
@@ -142,8 +133,9 @@ var _ = Describe("Storage", func() {
 				RunVMIAndExpectLaunch(vmi, false, 90)
 
 				expecter, err := tests.LoggedInCirrosExpecter(vmi)
-				Expect(err).To(BeNil())
+				Expect(expecter).NotTo(BeNil())
```
Comment (rmohr) [inline tests/storage_test.go]: We should not get an error AND an expecter? That is counterintuitive in the go world. Let's always close the expecter in `LoggedInCirrosExpecter` if such an error occurs where we already have the expecter.

File: tests/utils.go [hunk 2]
```diff
@@ -1165,8 +1165,9 @@ func LoggedInCirrosExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	resp, err := expecter.ExpectBatch(b, 180*time.Second)
 	if err != nil {
 		log.DefaultLogger().Object(vmi).Infof("Login: %v", resp)
+		return nil, err
```
Comment (rmohr) [inline tests/utils.go]: In this case you will have to close the expecter in here.

File: tests/utils.go [hunk 3]
```diff
@@ -1185,6 +1186,7 @@ func LoggedInAlpineExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	res, err := expecter.ExpectBatch(b, 180*time.Second)
 	if err != nil {
 		log.DefaultLogger().Object(vmi).Infof("Login: %v", res)
+		return nil, err
```
Comment (rmohr) [inline tests/utils.go]: You will have to close the expecter here.

Comment (cynepco3hahue) [reply tests/utils.go]: Oh did it under the previous commit, but did reset and forgot about it

**Pull request merged**